### ISSUE TITLE
ERROR: column p.proisagg does not exist

### ISSUE THREAD

**Issue by jflambert** [schemaspy/schemaspy#537]
Seeing a warning/error that I don't see with latest official release 6.0.0.

## Expected Behavior
Shouldn't see a warning, or warning should be more helpful.

## Current Behavior
Seeing a renderer warning with the latest schemaspy 6.1.0-SNAPSHOT

## Steps to Reproduce (for bugs)
1. java -jar ./schemaspy-6.1.0-SNAPSHOT.jar -o test -t pgsql -dp ./postgresql-42.2.5.jar -host localhost <your DB params>
2. Observe the following warning

```
INFO  - The following profiles are active: default
INFO  - Started Main in 2.867 seconds (JVM running for 3.844)
INFO  - Configuration file not found
INFO  - Starting schema analysis
INFO  - Connected to PostgreSQL - 11.2 (Ubuntu 11.2-1.pgdg18.04+1)
INFO  - Gathering schema details
Gathering schema details...................................................................................................................................................................WARN  - Failed to retrieve stored procedure/function details using sql 'select r.routine_name, case when p.proisagg then 'AGGREGATE' else 'FUNCTION' end as routine_type, case when p.proretset then 'SETOF ' else '' end || case when r.data_type = 'USER-DEFINED' then r.type_udt_name else r.data_type end as dtd_identifier, r.external_language as routine_body, r.routine_definition, r.sql_data_access, r.security_type, r.is_deterministic, d.description as routine_comment from information_schema.routines r left join pg_namespace ns on r.routine_schema = ns.nspname left join pg_proc p on ns.oid = p.pronamespace and r.routine_name = p.proname left join pg_description d on d.objoid = p.oid where r.routine_schema = :schema'
ERROR: column p.proisagg does not exist
  Hint: Perhaps you meant to reference the column "p.prolang".
  Position: 34
(3sec)
```

## Context
* Database type (e.g. MySQL): PostgreSQL 11.2
* Database driver version (is you use a special one): postgresql-42.2.5.jar
* Full command including arguments how you run SchemaSpy: See steps

## Your Environment
* Version used: schemaspy-6.1.0-SNAPSHOT.jar
* Operating System and version: Ubuntu 18.04
* Java version: java -version
openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4, mixed mode)

**Comment by npetzall** [comment]
Searching for p.proisagg among our issues (including closed) you'll find #470 

PostgreSQL made changes to their schemas in 11.2 so there is a new databaseType `-t psgsql11` that can be used.

**Comment by jflambert** [comment]
OK! Thanks. 

Could you maybe auto-detect that setting or improve the error message to suggest this new type?

**Comment by npetzall** [comment]
Auto-detect would be a cool feature I might look into that.

**Comment by jflambert** [comment]
Right, I would also argue that when PostgreSQL 12 is released, using a `-t pgsql11` option might be counter-intuitive.

**Comment by npetzall** [comment]
Man those 1 and l look the same.

We have taken the same approach for other dbms. So the idea is 11 or newer. 

Same goes for mssql and this is to not break backwards compatability.

**Comment by npetzall** [comment]
Documentation is needed for the strategy. It would be possible to check for databaseTypes that extends the selected to show alternatives.

Adding a version property to the databaseType would make a type of autodetection possible.

**Comment by npetzall** [comment]
A proposal for auto-detect #544

**Comment by donmedo2000** [comment]
Had thesame issue in phppgadmin and realised error is caused by phppgadmin directly accessing postgres schema using queries designed for postgres < 10. Work around is to run these two commands in ubuntu command prompt to effect column name change.

sed -i "s/NOT pp.proisagg/pp.prokind='f'/g" /usr/share/phppgadmin/classes/database/Postgres.php
sed -i "s/NOT p.proisagg/p.prokind='f'/g" /usr/share/phppgadmin/classes/database/Postgres.php

You may need to run command as sudo to avoid permission denied error. Also check to confirm correct case of phpPgAdmin folder name in above commands

**Comment by mmatela** [comment]
To prevent future confusion, maybe type `pgsql` should be an alias for `pgsql11` and there should be `pgsql10` for older versions (which occur less and less in the wild)?
